include/oneapi/dpl/pstl/omp/util.h

```diff
@@ -92,7 +92,7 @@ __should_run_serial(_Iterator __first, _Iterator __last) -> bool
     return __size <= static_cast<_difference_type>(__default_chunk_size);
 }
 
-template <typename _Index, std::enable_if_t<std::is_integral<_Index>::value, bool> = true>
+template <typename _Index, std::enable_if_t<std::is_integral_v<_Index>, bool> = true>
```

Comment:
We can probably switch this to `::std` right here for consistency.

Reply:
Actually, I see this entire `omp` directory uses `std::` as the convention. I think if we want use `::std` we should switch all instances. I have no strong preference either way.

Reply:
Agree, but not in this PR: I have separate branch for this purpose.